Automattic/pocket-casts-android
# [Manage Downloads] Allow users to dismiss low storage banner in download screen

## Description
- This PR will allow our users to dismiss the low storage banner from download screen for 14 days
- Context: p1734454637773259-slack-C07TL9YK9V1

Fixes #3380

## Testing Instructions
1. Apply [patch.patch](https://github.com/user-attachments/files/18201084/patch.patch)
2. Download an episode
3. Open downloads screen
4. See the low storage banner
5. Tap on the 3 dots button
6. Dismiss
7. Ensure the banner is hidden ✅
8. Wait more than 10 seconds (this is the mocked by the patch instead of 14 days)
9. Go to other screen
10. Back to Downloads screen
11. Ensure you see the banner back again ✅


## Screenshots or Screencast 
[Screen_recording_20241219_140315.webm](https://github.com/user-attachments/assets/26ac15f5-17f7-4fd6-b9a8-9825c9e56f74)


## Checklist
 
#### I have tested any UI changes...
